client/src/components/Desktop/Email/MailCommunications.js

```diff
@@ -0,0 +1,20 @@
+import React, { useContext } from 'react';
+import { MailNavigationContext } from '../../contexts/MailNavigationContext';
+import MailSenden from './MailSenden';
+import MailGesendet from './MailGesendet';
+import MailGelesen from './MailGelesen';
+import MailUngelesen from './MailUngelesen';
+
+export default function MailCommunications() {
+  const [activeNavigation] = useContext(MailNavigationContext);
+
+  if (activeNavigation === 'senden') {
+    return <MailSenden />;
```

Comment:
I find the English/German mix a little strange, but I can't think of a better solution right now